parts/linux/cloud-init/artifacts/mariner/cse_install_mariner.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+echo "Sourcing cse_install_distro.sh for Mariner"
+
+removeContainerd() {
+    retrycmd_if_failure 10 5 60 dnf remove -y moby-containerd
+}
+
+installDeps() {
+    apt_get_update || exit $ERR_APT_UPDATE_TIMEOUT
+    apt_get_dist_upgrade || exit $ERR_APT_DIST_UPGRADE_TIMEOUT
+    for apt_package in ca-certificates cifs-utils cracklib ebtables ethtool fuse git iotop iproute ipset iptables jq pam nfs-utils socat sysstat traceroute util-linux xz zip; do
+      if ! apt_get_install 30 1 600 $apt_package; then
+        exit $ERR_APT_INSTALL_TIMEOUT
+      fi
+    done
+}
+
+installGPUDrivers() {
+    echo "GPU drivers not yet supported for Mariner"
+    exit $ERR_GPU_DRIVERS_INSTALL_TIMEOUT
+}
+
+installSGXDrivers() {
+    echo "SGX drivers not yet supported for Mariner"
+    exit $ERR_SGX_DRIVERS_START_FAIL
+}
+
+getMobyPkg() {
```

Comment:
do we have a caller for this in Mariner?

Reply:
if not we can remove it

Reply:
Nope, removing.